src/event_object_movement.c

```diff
@@ -2453,7 +2453,15 @@ u8 CameraObjectGetFollowedObjectId(void)
 
 void CameraObjectReset2(void)
 {
+#ifdef UBFIX
+    struct Sprite* cameraObject = FindCameraObject();
+
+    if (cameraObject == NULL)
+        return;
+    cameraObject->data[1] = 2;
+#else
     FindCameraObject()->data[1] = 2;
+#endif
```

Comment:
This can be simplified so that the `UBFIX` only contains the relevant change:

```suggestion
    struct Sprite *cameraObject = FindCameraObject();
#ifdef UBFIX
    if (camera == NULL)
        return;
#endif
    cameraObject->data[1] = 2;
```